tests/test_pca.py

```diff
@@ -84,10 +81,20 @@ def wrapper(a: np.ndarray) -> DaskArray:
 }
 
 
+def dask_compute(*adatas: AnnData, X: bool = False, pca: bool = True) -> AnnData:
+    """Helper to compute dask arrays in AnnData objects, e.g. before comparison."""
+    for adata in adatas:
+        if X and isinstance(adata.X, DaskArray):
+            adata.X = adata.X.compute()
+        if pca and isinstance(adata.obsm["X_pca"], DaskArray):
+            adata.obsm["X_pca"] = adata.obsm["X_pca"].compute()
+    return adatas
```

Comment:
I think we have `to_memory()` on the `AnnData` object for this purpose? Why do we need to check these elements?

Reply:
`np.array_equal` throws a warning when run on dask arrays.

I’ll try `to_memory`, great point!